tests/installation/logpackages.pm

```diff
@@ -19,6 +19,15 @@ use warnings;
 use base "y2logsstep";
 use testapi;
 
+sub use_ssh_serial_console {
+    console('sol')->disable;
+    select_console('root-ssh');
+    $serialdev = 'sshserial';
+    set_var('SERIALDEV', 'sshserial');
+    bmwqemu::save_vars();
+}
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).

Reply:
Got it, thank you for your great suggestion and comments.